frontend/src/components/common/form/SppButton.tsx

```diff
@@ -29,7 +29,7 @@ const SppButton: React.FC<ISppButtonProps> = ({
 
   const iconColor = inEnhancedReferralProcess
```

Comment:
I think this carried over from the old button... The colour doesn't actually change anymore

Reply:
I just checked, also the tooltip doesn't work anymore

Reply:
Oh you're right, I think I vaguely remember the old button lol, looks like it is not used anywhere but the test now! Will remove